### ISSUE TITLE
< and > from imported markdown files display as &lt; and &gt;

### ISSUE THREAD

**Issue by natemcmaster** [NuGet/NuGetGallery#4783]
The gallery does not display angle brackets in code blocks correctly in the package's documentation section comes from importing a markdown README file.

# Repro
1. Edit a package. E.g. https://www.nuget.org/packages/Yarn.MSBuild/1.0.1/Edit
1. Add this URL to the "Import Readme" => "URL" box: https://raw.githubusercontent.com/natemcmaster/Yarn.MSBuild/v1.0.1/README.md. This file contains this section
````md
**Package Manager Console in Visual Studio**
```
PM> Install-Package Yarn.MSBuild
```
````

# Expected result

![image](https://user-images.githubusercontent.com/2696087/31050746-5801a22a-a609-11e7-9d56-9b9b32c4a37c.png)

# Actual result
It displays brackets as `&gt;` and `&lt;`
![image](https://user-images.githubusercontent.com/2696087/31050743-4536ea92-a609-11e7-8234-9b7d7ab6ca8e.png)

**Comment by anangaur** [comment]
Linking the feature https://github.com/NuGet/NuGetGallery/issues/2280

**Comment by natemcmaster** [comment]
Thanks!

**Comment by i3arnon** [comment]
Still doesn't work in inlined code segments with \` (i.e. \`Hamster\<T\>\` => `Hamster&lt;T&gt;`)
For example: https://www.nuget.org/packages/AsyncUtilities/0.5.0#

**Comment by Pathoschild** [comment]
Screenshot of the inline code issue:
>![image](https://user-images.githubusercontent.com/230581/32388262-88634528-c09d-11e7-94a2-725fea14f375.png)

**Comment by anangaur** [comment]
Hmm.. Looks like we fixed it for the code blocks but not for inline code.

**Comment by chenriksson** [comment]
Thanks for reporting this. I've opened a separate issue (#4946) to track this.

**Comment by anangaur** [comment]
@chenriksson  Looks like we were working in parallel ;). I created https://github.com/NuGet/NuGetGallery/issues/4947
